scripts/release-github.js

```diff
@@ -28,9 +28,9 @@ const url = require('url')
 const fs = require('fs')
 const https = require('https')
 const releaseAssets = require('gh-release-assets')
-const { version } = require('../package.json')
+const { version } = require('../packages/rum/package.json')
```

Comment:
we would still have to decide how to version the release if we want to release two packages separately

Reply:
For now let's keep using `v3.0.0` and publish artefacts for the js-base. We can revisit this later

Reply:
Do you want to follow different conventions for js-base and js-core then? It will collide with other packages then, We have to follow some other convention. Thats why its better to do it via <packageName>@version as lerna does.